### PR TITLE
update CHANGELOG.md

### PR DESCRIPTION
**Pull request by rfomin**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,43 +1,116 @@
...
+  - Added quickstart cache option (from DSDA-Doom). See `quickstart_cache_tics` in woof.cfg.
+  - Added "Last" weapon key binding.
+  - Added "Cycle Through All Weapons" option.
+  - Added support for SSG key binding when using vanilla compatibility level.
+  - Improved peformance when handling many input events at once.
+  - Optimized input polling for older computers (thanks to @gendlin).
+
+* [Gamepad](https://github.com/fabiangreffrath/woof/wiki/Gamepad):
+  - Added custom weapon slots. The default configuration uses the d-pad and matches the Doom II Xbox 360 port.
+  - Added rumble support.
```
Comment (rfomin) [inline CHANGELOG.md:43]: @ceski-1 Could you mention support for custom weapons/sfx? Also, how about turning on the rumble by default?

**Comment by MrAlaux** [comment]
Should I revise the changelog, as usual?

**Comment by rfomin** [comment]
> Should I revise the changelog, as usual?

Sure, any help will be appreciated.

**Comment by MrAlaux** [comment]
https://github.com/fabiangreffrath/woof/pull/2026/commits/5725d8701ceb056a4bcb933d320695172aee2099, what do you think?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,18 +1,21 @@
 **New Features and Improvements**
 
 * Mod Support:
-  - Added support for ID24 `SKYDEFS` and intermissions.
-  - Added support for ID24 `SBARDEF` with additional widescreen layout and widgets. Remove Boom HUD.
-  - Added support for PNG graphics, textures, flats and sprites. Automatically convert PNG palette. Convert 32-bit PNG to 8-bit with color degradation (not recommended).
+  - ID24 features:
+    - `SBARDEF`, with additional widescreen layout and widgets; consequently, removed Boom HUD.
+    - `SKYDEFS`.
+    - `TRAKINFO` (remixed soundtrack from "Doom + Doom II").
```
Comment (rfomin) [inline CHANGELOG.md:7]: `TRAKINFO` is not part of the ID24 spec.

**Comment by rfomin** [comment]
> [5725d87](https://github.com/fabiangreffrath/woof/commit/5725d8701ceb056a4bcb933d320695172aee2099), what do you think?

Looks good, thank you.

File: CHANGELOG.md (hunk 2 above)
Comment (MrAlaux) [reply CHANGELOG.md]: My bad, fixed in https://github.com/fabiangreffrath/woof/pull/2026/commits/0afe4b73121ce40225da7d8b9285509b7b9cf776.

File: CHANGELOG.md (hunk 1 above)
Comment (ceski-1) [reply CHANGELOG.md]: https://github.com/fabiangreffrath/woof/commit/fcdec943c13a3865cfb8aa8d86ea7d91a1da4fa6 and https://github.com/fabiangreffrath/woof/pull/2026/commits/85a6fb761f5d59ba9e028817301a565426c8209f

**Comment by fabiangreffrath** [comment]
The changelog should have a "Regressions" section listing all the features that were dropped in favor of the SBARDEF adoption, e.g. blinking keys.

**Comment by rfomin** [comment]
> The changelog should have a "Regressions" section listing all the features that were dropped in favor of the SBARDEF adoption, e.g. blinking keys.

Done in https://github.com/fabiangreffrath/woof/pull/2026/commits/cff92dafff79f9ab6f7421d485161b0c0c65dee5

**Review by fabiangreffrath — APPROVED** [review]

**Pull request merged**